autoload/test/rust/cargonextest.vim

```diff
@@ -114,6 +114,17 @@ function! s:test_namespace(filename) abort
           echo
           let l:package = l:modules[idx]
           let l:modules = l:modules[idx+1:]
+          " use package name, if present
+          let l:section = ''
+          for line in readfile(l:cargo_toml)
+            if line =~ '\[.\+\]' | let l:section = line | endif
+              if l:section == '[package]' && line =~ 'name'
+              let l:package = matchstr(line, '"\zs[^"]*\ze"')
+
+              break
+            endif
+          endfor
```

Comment:
This was my attempt to read the package name from a `Cargo.toml` using only RegExp.

If anyone knows of a better way, please leave a suggestion. ;)

Reply:
I think it's fine to use RegExp to parse it as you have added tests around this